pkg/config/constants/constants.go

```diff
@@ -153,6 +153,10 @@ const (
 	// load balancer, such as an Istio Gateway, is terminating the TLS.
 	CertProviderNone = "none"
 
+	// AlwaysReject is a special internal annotation that is always rejected in the validation webhook. This is used for
+	// testing the validation webhook.
+	AlwaysReject = "internal.istio.io/always-reject"
```

Comment:
```suggestion
	AlwaysReject = "internal.istio.io/webhook-always-reject"
```

ignorable nit: make the purpose of the annotation itself more obvious by the name.